docs/platform/conditions/actions/send_email.json

```diff
@@ -0,0 +1,53 @@
+{
+  "properties": {
+    "action": {
+      "$id": "#root/conditions/items/action",
+      "title": "Action",
+      "type": "object",
+      "required": [
+        "type",
+        "action_params"
+      ],
+      "properties": {
+        "type": {
+          "$id": "#root/conditions/items/action/type",
+          "title": "Type",
+          "type": "string",
+          "enum": [
+            "send_email"
+          ]
+        },
+        "properties": {
+          "action_params": {
+            "$id": "#root/conditions/items/action/send_email/action_params",
```

Comment:
do any of these `$id` make sense anymore?

Reply:
removing these